client/my-sites/site-settings/stats.jsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import SectionHeader from 'components/section-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import QuerySiteRoles from 'components/data/query-site-roles';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteRoles } from 'state/site-roles/selectors';
+import {
+	isJetpackModuleActive,
+	isJetpackModuleUnavailableInDevelopmentMode,
+	isJetpackSiteInDevelopmentMode
+} from 'state/selectors';
+
+class SiteStats extends Component {
```

Comment:
Tiny naming nit - should this be called `JetpackSiteStats` since we don't have a stats settings panel for WPCOM sites?